Tools/clinic/clinic.py

```diff
@@ -1965,12 +1965,15 @@ class BufferSeries:
     e.g. o[-1] is an element immediately preceding o[0].
     """
 
-    def __init__(self):
+    def __init__(self) -> None:
         self._start = 0
-        self._array = []
+        self._array: list[_TextAccumulator] = []
         self._constructor = _text_accumulator
 
-    def __getitem__(self, i):
+    def __getitem__(
+            self,
+            i: int
+    ) -> _TextAccumulator:
```

Comment:
Why not write it in one line? It is still pretty short.

Reply:
Sure; I have not yet developed a preference when it comes to annotation style :)

Reply:
FWIW, I agree with Serhiy — I would probably put this all on one line — but I also don't really care much either way (life's too short to spend it arguing over code formatting preferences 😄)

Reply:
No, it isn't! 😆